### ISSUE TITLE
This code looks awesome!!...does it work with the open source version octave :)?

### ISSUE THREAD

**Issue by andrewcz** [francescosolera/LDCT#1]

**Comment by LoverPP** [comment]
how  to work in the new dataset?   can I learn the model?   thank you  for help!

**Comment by francescosolera** [comment]
@LoverPP, yes it does. Take a look at the DEMO.m code where we first train the tracker on an arbitrary sequence and then test the learnt model on a different video.

**Comment by LoverPP** [comment]
but the data_PETS09S2L2\trajectories  hasn't .txt file;   running error

**Comment by LoverPP** [comment]
data_PETS09S2L2 set is incomplete

**Comment by LoverPP** [comment]
I change name of trainset, but running error.

**Comment by francescosolera** [comment]
Yes, if you will first need to prepare your data through the [data degradation script](https://github.com/francescosolera/LDCT/blob/64376f5485c8e0a0c3c03675d93be9b85cb0287b/data_PETS09S2L1/trajectories/deg_4_training.m#L1) which is currently placed inside the PETS09S2L1 folder.

**Comment by LoverPP** [comment]
can you tell me how to do it ?   Can you give me  complete data_PETS09S2L2？  Thank you !
